### PR TITLE
Fixes #42 - Queue are auto deleted immediately when client loose conn…

### LINKED ISSUE #42: Queue are auto deleted immediately when client loose connection
**Describe the bug**
Queue are auto deleted immediately when client loose connection

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a clock
2. Disconnect client
3. queue is deleted immediately 

**Expected behaviour**
Queues should wait for expiry time before delete

### PR DESCRIPTION
**Pull request by rockem**
…ection

### CODE REVIEW

File: src/test/kotlin/e2e/test/io/ticktok/server/support/App.kt [hunk 1]
```diff
@@ -200,8 +194,8 @@ class App(profile: String) {
         resumeAllPausedClocks()
         sleep(500)
         val response = Request.Post(createAuthenticatedUrlFor("/api/v1/clocks/purge")).execute().returnResponse()
-        sleep(500)
         assertThat(response.statusLine.statusCode, `is`(204))
+        sleep(1000)
```
Comment (wanishing) [inline src/test/kotlin/e2e/test/io/ticktok/server/support/App.kt]: why? seemed a bit weird

File: src/test/kotlin/e2e/test/io/ticktok/server/CoreE2ETest.kt [hunk 2]
```diff
@@ -51,7 +51,7 @@ class CoreE2ETest : CommonAppE2ETest() {
         }
 
         @Test
-        fun purgeClocks() {
+        fun purgeRedundantClocks() {
```
Comment (wanishing) [inline src/test/kotlin/e2e/test/io/ticktok/server/CoreE2ETest.kt]: what's the meaning here of 'redundant'?

File: src/test/kotlin/e2e/test/io/ticktok/server/support/App.kt (hunk 1 above)
Comment (wanishing) [reply src/test/kotlin/e2e/test/io/ticktok/server/support/App.kt]: to sleep on the end of the function. the sleep is for the callers of this function?

File: src/test/kotlin/e2e/test/io/ticktok/server/CoreE2ETest.kt (hunk 2 above)
Comment (wanishing) [reply src/test/kotlin/e2e/test/io/ticktok/server/CoreE2ETest.kt]: so why were they created on the first place?
the word "redundant" is confusing

File: src/test/java/test/io/ticktok/server/tick/rabbit/RabbitTickChannelOperationsTest.java [hunk 3]
```diff
@@ -74,27 +70,37 @@ private Object receivedTick() {
...
     }
 
     @Test
     void ignoreEnableFailureIfQueueNotExists() {
         tickChannelExplorer.enable(new Clock("papa", "every.321.seconds"));
         // pass
     }
+
+    @Test
+    void shouldNotDeleteQueueImmediatelyAfterClientDisconnect() {
```
Comment (wanishing) [inline src/test/java/test/io/ticktok/server/tick/rabbit/RabbitTickChannelOperationsTest.java]: where can I see here the client had disconnected?

File: src/test/kotlin/e2e/test/io/ticktok/server/CoreE2ETest.kt (hunk 2 above)
Comment (rockem) [reply src/test/kotlin/e2e/test/io/ticktok/server/CoreE2ETest.kt]: renamed

**Pull request merged**